cqtypes/inet.go

```diff
@@ -0,0 +1,158 @@
+//nolint:revive
+package cqtypes
+
+import (
+	"encoding"
+	"fmt"
+	"net"
+	"strings"
+)
+
+// Network address family is dependent on server socket.h value for AF_INET.
+// In practice, all platforms appear to have the same value. See
+// src/include/utils/inet.h for more information.
+const (
+// defaultAFInet  = 2
+// defaultAFInet6 = 3
+)
```

Comment:
Looks like this constant block is now unused 🤔

Reply:
yes. removed. it is used in pgx to encode to postgresql binary format but we don't need it as we just need the decode part and not from postgresql